projects/core/src/auth/user-auth/services/oauth-lib-wrapper.service.ts

```diff
@@ -22,15 +23,35 @@ import { AuthConfigService } from './auth-config.service';
 })
 export class OAuthLibWrapperService {
   events$: Observable<OAuthEvent> = this.oAuthService.events;
+  currentBaseSite: BaseSite;
 
   // TODO: Remove platformId dependency in 4.0
   constructor(
     protected oAuthService: OAuthService,
     protected authConfigService: AuthConfigService,
     @Inject(PLATFORM_ID) protected platformId: Object,
-    protected winRef: WindowRef
+    protected winRef: WindowRef,
+    protected baseSiteService?: BaseSiteService
   ) {
     this.initialize();
+    if (baseSiteService) {
+      baseSiteService
+        .get()
+        .pipe(take(1))
+        .subscribe((site) => {
+          if (site?.cdcSiteConfig && site?.uid) {
```

Comment:
only MCS has cdcSiteConfig, with this we can set oidc config and don't need to add feature flag.